### PR TITLE
feat: Implement CLIENT KILL

### PR DESCRIPTION
**Pull request by siyopao**
Part of #174.

### CODE REVIEW

**Comment by siyopao** [comment]
Hello @uki00a! I have attempted to implement CLIENT KILL, but I am having a bit of trouble writing the tests. I can see that I have kill a client, and that I can no longer use the client, but when I check if the client `isClosed` I get `false`. Is it enough to check that a client is killed? Any advice? Thanks!

**Comment by uki00a** [comment]
Hi @siyopao, Thank you for working on this PR!

> Is it enough to check that a client is killed?

Yes, it is enough to check that a client is killed for now. :ok_man:

**Comment by siyopao** [comment]
Okay! Thanks. I update to just check if the client is killed. The tests are not exhaustive (no laddr or user) but easy enough if you would like me to add them in.

**Comment by siyopao** [comment]
Hi again! Sorry, one more question: here I am also not checking for an absence of client to kill. Should I, or just let Redis return an error and let that bubble up? Thanks!

**Comment by uki00a** [comment]
> Should I, or just let Redis return an error and let that bubble up?"

Yes, I think this is OK. :ok_man:

File: command.ts [hunk 1]
```diff
@@ -51,8 +51,17 @@ export interface BitfieldWithOverflowOpts extends BitfieldOpts {
 
 export type ClientCachingMode = "YES" | "NO";
```
Comment (uki00a) [inline command.ts:53]: Please open another PR for this change. Other than that, it looks good to me. :+1:

**Review by uki00a — CHANGES_REQUESTED** [review]

File: command.ts (hunk 1 above)
Comment (siyopao) [reply command.ts:53]: Sure, no problem. I created #214. After that is merged I can rebase this branch. Thanks!

**Review by uki00a — APPROVED** [review]
LGTM! :ok_man:

**Pull request merged**